src/DataGrid.tsx

```diff
@@ -1189,6 +1153,27 @@ function DataGrid<R, SR, K extends Key>(
           onKeyDown={handleKeyDown}
         />
       )}
+      {scrollToPosition !== null && (
+        <ScrollToCell
+          scrollToPosition={scrollToPosition}
+          onScroll={() => {
+            if (scrollToPosition.idx !== undefined) {
+              const { key, width } = columns[scrollToPosition.idx];
+              if (
+                // with auto-sized column we need to keep scrolling
+                // until the column width is set and it is in the viewport
+                typeof width === 'string' &&
```

Comment:
they will but I think if the width is set then we can safely assume the cell is visible 🤔

Reply:
Imagine this scenario:
- you've jumped to the end of the grid
- you `scrollToCell` to a column on the left
- scrolling to it would render other columns on its cell that may not have been sized before, or were invalidated